### PR TITLE
odim 3736:Updating log during publishing the task event 

### PR DESCRIPTION
**Pull request by jeevan-kamkar**

### CODE REVIEW

**Review by tamanna-k — APPROVED** [review]

**Review by skhan-anjum — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]
LGTM

File: svc-task/tmessagebus/publish.go [hunk 1]
```diff
@@ -59,6 +59,5 @@ func Publish(taskURI string, messageID string, eventType string) {
 		log.Println("Unable Publish events to kafka", err)
```
Comment (Bharath-KKB) [inline svc-task/tmessagebus/publish.go]: change the message

File: svc-task/tmessagebus/publish.go [hunk 2]
```diff
@@ -59,6 +59,5 @@ func Publish(taskURI string, messageID string, eventType string) {
...
 		return
 	}
-	log.Printf("debug: %s Event Published", messageID)
-
+	log.Println("info: TaskURI:", taskURI, ", EventID:", eventID, ", MessageID:", messageID, " of the published event")
```
Comment (Bharath-KKB) [inline svc-task/tmessagebus/publish.go]: provide sample logs after this change

Comment (Bharath-KKB) [inline svc-task/tmessagebus/publish.go]: what is the log level log.Println will print here ? "info: " should not be there.

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
address comments

File: svc-task/tmessagebus/publish.go (hunk 2 above)
Comment (Bharath-KKB) [reply svc-task/tmessagebus/publish.go]: remove "of the published event"
In the following what is **'I'** 
2020-11-10 08:01:05.810994 **I** | info: TaskURI: /redfish/v1/TaskService/Tasks/taske5838831-46f3-44da-be7f-8434e36c8b16 , EventID: 8f80c321-5781-4ac2-a1c5-7ee04ad0feb0 , MessageID: TaskEvent.1.0.1.TaskCompletedOK  of the published event

Comment (Bharath-KKB) [reply svc-task/tmessagebus/publish.go]: remove **of the published event"** from the log

Comment (jeevan-kamkar) [reply svc-task/tmessagebus/publish.go]: updated

File: svc-task/tmessagebus/publish.go (hunk 1 above)
Comment (jeevan-kamkar) [reply svc-task/tmessagebus/publish.go]: updated

**Review by skhan-anjum — APPROVED** [review]

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**